src/controller/python/chip-device-ctrl.py

```diff
@@ -629,17 +624,73 @@ def emptyline(self):
 device_manager = DeviceMgrCmd(rendezvousAddr=None,
                              controllerNodeId=0, bluetoothAdapter=0)
 
-
 # CHIP commands needed by the Harness Tool
 def echo_alive(message):
     print(message)
     return message
 
-def ble_scan() -> Dict[Any, Any]:
+
+def resolve(fabric_id: int, node_id: int) -> Dict[str, Any]:
     try:
         __check_supported_os()
-        device_manager.do_blescan("")
+        err = device_manager.devCtrl.ResolveNode(fabric_id, node_id)
+        if err == 0:
+            address = device_manager.devCtrl.GetAddressAndPort(int(args[1]))
+            address = "{}:{}".format(
+                *address) if address else "unknown"
```

Comment:
if address "unknown" don't we need to send back  __get_response_dict(status = StatusCodeEnum.FAILED, error = str(ex)) ?

Reply:
Resolved